src/ServiceStack/ServerEventsFeature.cs

```diff
@@ -1324,6 +1326,9 @@ async Task DoAsyncTasks(CancellationToken token = default)
 
                     if (NotifyChannelOfSubscriptions && sub.Channels != null && NotifyLeaveAsync != null)
                         await NotifyLeaveAsync(sub).ConfigAwait();
+                    
+                    if (OnRemoveSubscriptionAsync != null)
+                        OnRemoveSubscriptionAsync(sub).ConfigAwait();
```

Comment:
need to `await` here otherwise PR LGTM 👍

Reply:
Ahh good catch, added.